README.md

```diff
@@ -32,7 +32,7 @@ Try our [interactive tutorial].
 Take a free course on [Scalable Microservices with Kubernetes].
 
 To use Kubernetes code as a library in other applications, see the [list of published components](https://git.k8s.io/kubernetes/staging/README.md).
-Use of the `k8s.io/kubernetes` module or `k8s.io/kubernetes/...` packages as libraries is not supported.
+Use of the `k8s.io/kubernetes` module or `k8s.io/kubernetes/...` packages as libraries are not supported.
```

Comment:
this is incorrect

Reply:
How? I’ve always thought that `are` was used after plural nouns. `packages as libraries` is plural. Or am I missing something?

Reply:
It's understandably tricky. In this context, the verb "is" refers to the noun "Use" and not to "libraries". If you strip out the extraneous parts, the skeleton of this sentence is "Use is not supported".

Reply:
Ahhh, I see it now. Thank you! 😄